package.json

```
@@ -13,9 +13,10 @@
     "postinstall": "manypkg check",
     "clean:packages": "yarn workspaces run clean",
     "build": "tsc -b ./tsconfig.monorepo.json",
-    "test": "jest",
-    "test:core": "npm test --prefix packages/core",
-    "dev": "node ./scripts/dev.js",
+    "build:dev": "tsc -b ./tsconfig.monorepo.json -w",
```

IMHO this would be better as `dev` is already "reserved" for the "dev" script and this has no direct relation to it (even though the dev script is kinda using it already)

```suggestion
    "build:watch": "tsc -b ./tsconfig.monorepo.json -w",
```

---

This was reverted